pdr_backend/accuracy/app.py

```diff
@@ -122,9 +122,9 @@ def aggregate_statistics(
         and the total counts of correct predictions and slots evaluated.
     """
 
-    total_staked_yesterday = (
-        total_staked_today
-    ) = total_correct_predictions = total_slots_evaluated = 0
+    total_staked_yesterday = total_staked_today = total_correct_predictions = (
+        total_slots_evaluated
+    ) = 0
```

Comment:
Why is this here? Better to be in a separate change. Avoids confusing Mustafa / Roberto / others.